libcob/profiling.c

```diff
@@ -336,7 +336,7 @@ print_monotonic_time (FILE *file, cob_ns_time t) {
 	if (seconds > 1000) {
 		fprintf (file, "%d s", seconds);
 	} else {
-		fprintf (file, "%d.%03Ld s", seconds, milliseconds);
+		fprintf (file, "%d.%03d s", seconds, (unsigned int)milliseconds);
```

Comment:
As far as I can tell, "%Ld" does not exist. This caused crashes under MSVC.

Reply:
I'll fix this one with the next commit